hasura.planx.uk/migrations/1679669504707_create_table_public_payment_requests/up.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE "public"."payment_requests" (
+  "payment_request_id" uuid NOT NULL DEFAULT gen_random_uuid(),
+  "session_id" uuid NOT NULL,
+  "created_at" timestamptz NOT NULL DEFAULT now(),
+  "payee_email" text NOT NULL,
+  "session_preview_data" jsonb NOT NULL,
```

Comment:
Maybe this is too much future proofing, but have a JSONB would be more generic and non-planning specific. This would mean that a council could spin up a service with request to pay outside of the planning context.

I appreciate that currently we're going to be hardcoding what gets written to this column as part of the API, so happy to go with columns if that makes more sense?

Reply:
The implementation and type checks are all expecting a set of defined values which feels appropriate at this point in my view. Happy to go with something more flexible if you'd prefer but I'd personally lean in favor of more explicit values here (especially since migrations are so easy).